src/openforms/submissions/models/submission.py

```diff
@@ -182,6 +182,13 @@ class Submission(models.Model):
             "Indication whether the registration in the configured backend was successful."
         ),
     )
+    pre_registration_completed = models.BooleanField(
+        _("pre-registration completed"),
+        default=False,
+        help_text=_(
+            "Indicates whether the pre-registration task completed successfully."
+        ),
+    )
```

Comment:
we'll need a data migration to update existing records that have a succesful registration I think? for data consistency sake.

I think we can also lock this in at the DB level with a constraint that prevents the `registration_status=completed, pre_registration_completed=False` combination as it functionally does not make any sense, right?